tests/user_commands/tests.py

```diff
@@ -247,6 +247,22 @@ def test_mutually_exclusive_group_required_options(self):
         with self.assertRaisesMessage(CommandError, msg):
             management.call_command('mutually_exclusive_required', stdout=out)
 
+    def test_mutually_exclusive_group_required_with_boolean(self):
+        tests = (
+            ('flag_true', True),
+            ('flag_false', False),
+            ('const', 42),
```

Comment:
I think we should pass `True` in all cases, because we're selecting a flag not declaring a value.